.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -5,19 +5,34 @@ about: Report a problem with rclone
 
 <!--
 
-Welcome :-) We understand you are having a problem with rclone; we want to help you with that!
+We understand you are having a problem with rclone; we want to help you with that!
 
-If you've just got a question or aren't sure if you've found a bug then please use the rclone forum:
+**STOP and READ**
+**YOUR POST WILL BE REMOVED IF IT IS LOW QUALITY**:
+Please show the effort you've put in to solving the problem and please be specific.
+People are volunteering their time to help! Low effort posts are not likely to get good answers!
+
+If you've just got a question or suspect you've found a bug then please use the rclone forum:
 
     https://forum.rclone.org/
 
-instead of filing an issue for a quick response.
+with the "Help & Support Template" for a quick response instead of filing an issue on this repo.
 
-If you think you might have found a bug, please can you try to replicate it with the latest beta?
+If you think you might have found a bug, try to replicate it with the latest release or latest beta.
+Update your rclone:
 
-    https://beta.rclone.org/
+    rclone self-update
+or
+    rclone self-update --beta
```

Comment:
Do you think we should add something like

If your rclone reports `unknown command "self-update"` then you'll need to update manually following the [install instructions](https://rclone.org/install/)

?

PS I think the install instructions should probably mention self-update too :-)

Reply:
> I think the install instructions should probably mention self-update too

Please review https://github.com/rclone/rclone/pull/5122 :pray: